.github/workflows/raedar_tests CI/raedar.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: raedar tests
+
+on:
+  push:
+    branches: [ master,develop ]
+  pull_request:
+    branches: [ master,develop ]
```

Comment:
Look for a way of selecting all the branches.

Reply:
Yeah, I agree with @huxaiphaer, the jobs should only happen when the attempt is to develop.

Reply:
I have removed master branch from the list